fahrschein/src/main/java/org/zalando/fahrschein/ReaderManager.java

```diff
@@ -0,0 +1,10 @@
+package org.zalando.fahrschein;
+
+import org.zalando.fahrschein.domain.Subscription;
+
+import java.util.Optional;
+
+@FunctionalInterface
+public interface ReaderManager {
```

Comment:
Thanks for the comment @jhorstmann.
I'll follow your advice and make the changes you mentioned.